MyProject/src/main/java/com/netcracker/database/DBConfiguration.java

```diff
@@ -0,0 +1,19 @@
+package com.netcracker.database;
+
+import com.arangodb.ArangoDB;
+import com.arangodb.springframework.annotation.EnableArangoRepositories;
+import com.arangodb.springframework.config.AbstractArangoConfiguration;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableArangoRepositories(basePackages = {"com.netcracker"})
+public class DBConfiguration extends AbstractArangoConfiguration {
+    @Override
+    public ArangoDB.Builder arango() {
+        return new ArangoDB.Builder().host("localhost", 8529).user("root").password("root");
```

Comment:
arango host, port, user and password should not be hardcoded, these parameters should be configurable through the application.properties. Btw please use application.yaml instead of properties.